rpc/src/v1/types/transaction.rs

```diff
@@ -51,6 +51,9 @@ pub struct Transaction {
 	pub creates: Option<H160>,
 	/// Raw transaction data
 	pub raw: Bytes,
+	/// Public key of the signer.
+	#[serde(rename="publicKey")]
+	pub public_key: Option<H512>,
```

Comment:
This transaction structure diverges from the spec: https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_gettransactionbyhash

Affected: all `getTransactionBy*` and `getBlockBy*` functions